db-scripts/10_initial_table_load.sql

```diff
@@ -1,158 +1,1124 @@
-,(104,'Cubone','Ground','',320,50,50,95,40,50,35,1,0)
-,(105,'Marowak','Ground','',425,60,80,110,50,80,45,1,0)
-,(106,'Hitmonlee','Fighting','',455,50,120,53,35,110,87,1,0)
-,(107,'Hitmonchan','Fighting','',455,50,105,79,35,110,76,1,0)
-,(108,'Lickitung','Normal','',385,90,55,75,60,75,30,1,0)
-,(109,'Koffing','Poison','',340,40,65,95,60,45,35,1,0)
-,(110,'Weezing','Poison','',490,65,90,120,85,70,60,1,0)
-,(111,'Rhyhorn','Ground','Rock',345,80,85,95,30,30,25,1,0)
-,(112,'Rhydon','Ground','Rock',485,105,130,120,45,45,40,1,0)
-,(113,'Chansey','Normal','',450,250,5,5,35,105,50,1,0)
-,(114,'Tangela','Grass','',435,65,55,115,100,40,60,1,0)
-,(115,'Kangaskhan','Normal','',490,105,95,80,40,80,90,1,0)
-,(116,'Horsea','Water','',295,30,40,70,70,25,60,1,0)
-,(117,'Seadra','Water','',440,55,65,95,95,45,85,1,0)
-,(118,'Goldeen','Water','',320,45,67,60,35,50,63,1,0)
-,(119,'Seaking','Water','',450,80,92,65,65,80,68,1,0)
-,(120,'Staryu','Water','',340,30,45,55,70,55,85,1,0)
-,(121,'Starmie','Water','Psychic',520,60,75,85,100,85,115,1,0)
-,(122,'Mr. Mime','Psychic','Fairy',460,40,45,65,100,120,90,1,0)
-,(123,'Scyther','Bug','Flying',500,70,110,80,55,80,105,1,0)
-,(124,'Jynx','Ice','Psychic',455,65,50,35,115,95,95,1,0)
-,(125,'Electabuzz','Electric','',490,65,83,57,95,85,105,1,0)
-,(126,'Magmar','Fire','',495,65,95,57,100,85,93,1,0)
-,(127,'Pinsir','Bug','',500,65,125,100,55,70,85,1,0)
-,(128,'Tauros','Normal','',490,75,100,95,40,70,110,1,0)
-,(129,'Magikarp','Water','',200,20,10,55,15,20,80,1,0)
-,(130,'Gyarados','Water','Flying',540,95,125,79,60,100,81,1,0)
-,(131,'Lapras','Water','Ice',535,130,85,80,85,95,60,1,0)
-,(132,'Ditto','Normal','',288,48,48,48,48,48,48,1,0)
-,(133,'Eevee','Normal','',325,55,55,50,45,65,55,1,0)
-,(134,'Vaporeon','Water','',525,130,65,60,110,95,65,1,0)
-,(135,'Jolteon','Electric','',525,65,65,60,110,95,130,1,0)
-,(136,'Flareon','Fire','',525,65,130,60,95,110,65,1,0)
-,(137,'Porygon','Normal','',395,65,60,70,85,75,40,1,0)
-,(138,'Omanyte','Rock','Water',355,35,40,100,90,55,35,1,0)
-,(139,'Omastar','Rock','Water',495,70,60,125,115,70,55,1,0)
-,(140,'Kabuto','Rock','Water',355,30,80,90,55,45,55,1,0)
-,(141,'Kabutops','Rock','Water',495,60,115,105,65,70,80,1,0)
-,(142,'Aerodactyl','Rock','Flying',515,80,105,65,60,75,130,1,0)
-,(143,'Snorlax','Normal','',540,160,110,65,65,110,30,1,0)
-,(144,'Articuno','Ice','Flying',580,90,85,100,95,125,85,1,1)
-,(145,'Zapdos','Electric','Flying',580,90,90,85,125,90,100,1,1)
-,(146,'Moltres','Fire','Flying',580,90,100,90,125,85,90,1,1)
-,(147,'Dratini','Dragon','',300,41,64,45,50,50,50,1,0)
-,(148,'Dragonair','Dragon','',420,61,84,65,70,70,70,1,0)
-,(149,'Dragonite','Dragon','Flying',600,91,134,95,100,100,80,1,0)
-,(150,'Mewtwo','Psychic','',680,106,110,90,154,90,130,1,1)
-,(151,'Mew','Psychic','',600,100,100,100,100,100,100,1,0);
+INSERT INTO pokemon (ndex, name, type_1, type_2, total, hp, attack, defense, sp_attack, sp_defense, speed, generation, is_legendary)

```

Comment:
I think ndex should be index

Reply:
@phantomfirefox - ndex was referred in other sources as "national pokemon index" as there could technically be another index per gen/game combo. I'd steer clear of "index" as to appease the SQL reserved word gods but happy to come up w/another name.

Reply:
Ohhh, I see. That does make sense and leave as `ndex` as national dex. I just thought the `i` was missing in looking it over quickly. We can keep as is here